zilencer/views.py

```diff
@@ -682,6 +696,10 @@ def update_remote_realm_data_for_server(
     # Update RemoteRealm entries, for which the corresponding realm's info has changed
     # (for the attributes that make sense to sync like this).
     for remote_realm in already_registered_remote_realms:
+        # TODO: We'll also want to check if .realm_locally_deleted is True, and if so,
+        # toggle it off (and potentially restore registraton_deactivated=True too),
+        # since the server is now sending us data for this realm again.
```

Comment:
Just living a `TODO` for now, since I'm not sure how it should handle `registraton_deactivated`. We don't yet have examples of toggling `registration_deactivated` on/off in the codebase i think, so possibly this should wait for decision on details of how that attr will be used?

Reply:
Yeah I think maybe we can wait until we see what realms actually get this tag and respond accordingly.